flashcards/backend/decks/models.py

```diff
@@ -1,3 +1,20 @@
 from django.db import models
 
-# Create your models here.
+
+class Category(models.Model):
+    """Model for representing Category for deck.
+    :param name: Category name.
+    :type name: str
+    :param description: Description for category.
+    :type description: str
+    """
+
+    name = models.CharField(max_length=24, unique=True)
```

Comment:
```
root@243b4ca253cf:/app/decks# mypy .
models.py:12: error: Need type annotation for "name"  [var-annotated]
models.py:13: error: Need type annotation for "description"  [var-annotated]
```

Reply:
I argue here, because I believe this is a mypy issues. This error is pretty inconsistent. In some files it says that there is type annotation error for some fields and in other files mypy says all is good with same fields.

Reply:
running `mypy .` in root directory is a problem. We should scan files added to the repository

azeno/quiz/player/models.py has the same type of error, so mypy works good
```
root@64c05e454aa9:/app/players# mypy models.py 
models.py:19: error: Need type annotation for "uuid"  [var-annotated]
models.py:22: error: Need type annotation for "nick"  [var-annotated]
models.py:23: error: Need type annotation for "rank"  [var-annotated]
models.py:26: error: Need type annotation for "is_bot"  [var-annotated]
models.py:31: error: Need type annotation for "is_active"  [var-annotated]
models.py:34: error: Need type annotation for "created_at"  [var-annotated]

```

Reply:
There is a recording where this error occured only when mypy was run in app directory and when it was run in root directory it didn't throw any errors. So I still think this error inconsistent.